src/data-types/datetime2.ts

```diff
@@ -80,7 +81,7 @@ module.exports = {
         buffer.writeUInt24LE(Math.floor((+parameter.value - UTC_YEAR_ONE) / 86400000));
       } else {
         const dstDiff = -(parameter.value.getTimezoneOffset() - YEAR_ONE.getTimezoneOffset()) * 60 * 1000;
-        buffer.writeUInt24LE(Math.floor((+parameter.value - YEAR_ONE + dstDiff) / 86400000));
+        buffer.writeUInt24LE(Math.floor((+parameter.value - +YEAR_ONE + dstDiff) / 86400000));
```

Comment:
@arthurschreiber I just wanted to double check with you if this change is still valid. Without the '+', TypeScript shows an error

Reply:
Yeah, this is correct. It's not an error in JavaScript itself, because JavaScript would automatically perform the conversion from `Date` to `number`, but TypeScript wants us to be explicit about this (to help catch potential bugs due to typos).